oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/query/AggregationQueryEsDAO.java

```diff
@@ -120,11 +120,13 @@ public List<SelectedRecord> sortMetrics(final TopNCondition condition,
         final List<Map<String, Object>> buckets =
             (List<Map<String, Object>>) idTerms.get("buckets");
         for (Map<String, Object> termsBucket : buckets) {
-            SelectedRecord record = new SelectedRecord();
-            record.setId((String) termsBucket.get("key"));
             Map<String, Object> value = (Map<String, Object>) termsBucket.get(valueColumnName);
-            record.setValue(String.valueOf(((Number) value.get("value")).longValue()));
-            topNList.add(record);
+            if (value.get("value") != null) {
```

Comment:
This could only avoid NPE. I think @wankai123 has found the root cause, should be able to fix soon.